cmd/serve.go

```diff
@@ -311,7 +311,8 @@ func (c *ServeCommand) Command(serverService ServerServiceInterface, monitorServ
 			OptType:   types.String,
 			ConfigKey: &adminServeOpts.AdminApiKey,
 			Required:  true,
-		})
+		}, cmdUtils.TenantXLMBootstrapAmount(&adminServeOpts.TenantAccountNativeAssetBootstrapAmount),
```

Comment:
Can we add this configuration to the docker-compose and helmchart ?

Reply:
sure!